src/field_theory/krull_topology.lean

```diff
@@ -287,3 +287,71 @@ begin
 end
 
 end totally_disconnected
+
+
+section finite
```

Comment:
All of this seems to be in the root namespace. Some of the names are too generic for that. Maybe wrap this in `namespace krull_topology`?